crates/subspace-service/Cargo.toml

```diff
@@ -21,6 +21,7 @@ derive_more = "0.99.17"
 frame-support = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate", rev = "c4f3d028621edb293d2c423516221aa396f76a2d" }
 futures = "0.3.21"
 jsonrpc-core = "18.0.0"
+pallet-feeds = { version = "0.1.0", default-features = false, path = "../pallet-feeds" }
```

Comment:
As described in another comment, we don't depend on pallets for runtime APIs anywhere and it would be preferred to keep it that way.